source/fundamentals/crud/run-command.txt

```diff
@@ -15,46 +15,50 @@ Run a Command
 Overview
 --------
 
-In this guide, you can learn how to run a **database command** with the
+In this guide, you can learn how to run a database command with the
 {+driver-short+}. You can use database commands to perform a variety of
 administrative and diagnostic tasks, such as fetching server statistics,
 initializing a replica set, or running an aggregation pipeline.
 
-Compose a Command
+Execute a Command
 -----------------
 
-The {+driver-short+} provides the following methods to run database
-commands: 
+To run a database command, you must specify the command and any relevant
+parameters in a command document, then pass the command document to a
+wrapper method. The command document must be an order-preserving type
+such as ``bson.D``. The {+driver-short+} provides the following methods
+to run database commands:
 
 - ``RunCommand()``, which returns the command response as a
   ``SingleResult`` type. You can use this method with any database command.
 - ``RunCommandCursor()``, which returns the command response as a
   ``Cursor`` type. You can use this method if your database command
   returns multiple result documents.
 
-Each method requires a Context and a command document. The
-command document must be an order-preserving type such as ``bson.D``.
-The following code shows how you can use the ``RunCommandCursor()``
-method to run the ``listCollections`` command on a database:
+The following code shows how you can use the ``RunCommand()``
+method to run the ``hello`` command, which returns information about
+the current member's role in the replica set, on a database:
 
 .. code-block:: go
 
-   command := bson.D{{"listCollections", 1}}   
-   cursor, err := db.RunCommandCursor(context.TODO(), command)
+   command := bson.D{{"hello", 1}}
 
-For a full list of database commands, see the :ref:`Additional
-Information <addl-info-runcommand>`.
+   var result bson.M
+   err = db.RunCommand(context.TODO(), command).Decode(&result)
+
+For a full list of database commands and corresponding parameters, see
+the :ref:`Additional Information section <addl-info-runcommand>`.
 
 .. note:: Read Preference
 
    ``RunCommand()`` and ``RunCommandCursor()`` do not obey the read
-   preference set for the database. You can set a read preference for a
-   database command by passing a ``RunCmdOptions`` type to either method:
+   preference you set on your ``Database`` object. You can set a read preference for a
```

Comment:
Issue: If someone else has set the read preference in a different way, does ``RunCommand()`` obey that? As written, it sounds like it will ignore just the one that I set.

Suggestion: Clarify scope of what ``RunCommand()`` will respect.

Reply:
What this means is that when you instantiate a database object, you can set a read preference. But, the RunCommand method(s) will not obey that read preference.
I think the change I made improves clarity. LMK